### DISCUSSION TITLE
Base styles injected into index.html when using postcss plugin

### DISCUSSION THREAD

**Discussion by rhythnic** [tailwindlabs/tailwindcss#12411] [Help] [answered]
I am using a content security policy that forbids any style tags.

```
<meta http-equiv="Content-Security-Policy" content="default-src 'self'; script-src 'self'" />
```

I have integrated tailwindcss with preact-cli by using the tailwindcss postcss plugin.  I haven't pinpointed the cause yet, but when using `@tailwind base` in the main css file, the base styles are included in the css bundle, but they're also injected in a style tag in index.html.  When I remove `@tailwind base`, the style tag is not in the built index.html.

Is there a configuration variable that would prevent the injection of the base styles into index.html?  Or is this not the domain of the tailwind postcss plugin, and maybe I could prevent it via the preact-cli configuration?

**Comment by wongjn** [comment, marked as answer]
It doesn't seem like this would be in the domain of the Tailwind PostCSS plugin. I would expect you'd want to configure this in preact-cli.

**Reply by rhythnic** [reply]
Yep.  Adding `--no-inline-css` to the preact cli command fixed it.
